### ISSUE TITLE
channel.setTopic(); cannot be used to set a topic to nothing

### ISSUE THREAD

**Issue by gt-c** [discordjs/discord.js#2407]
```js
channel.setTopic("");
// will refuse to set the topic, as with
channel.setTopic(null);

/*- discord.js version: stable
- node.js version: v9.7.1
- Operating system: Windows 10
note: issue is _only_ on stable*/

**Comment by cdemirer** [comment]
I could also reproduce this with:
discord.js version: 11.3.2 (stable) e5bd6ec150baee5ee4ca0830b80753b7c59f4844
node.js version: 8.4.0
OS: macOS 10.12.6

I think I found the problem: At [line 336 of RESTMethods.js](https://github.com/discordjs/discord.js/blob/e5bd6ec150baee5ee4ca0830b80753b7c59f4844/src/client/rest/RESTMethods.js#L336), empty string evaluates to false and ignored.

**Comment by Lewdcario** [comment]
Should be fixed with 92b421607e409711ce05260acd54078a59ea368a, which you can install via the `11.3-dev` branch.
